README.md

```diff
@@ -38,10 +38,40 @@ This repository holds the notebooks themselves, but in a harder-to-read unexecut
     * [Intermediate: Create TESS FFI Cutout using Python Requests](https://spacetelescope.github.io/notebooks/notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.html)
 
 
-### Building Locally
+## Building Locally
+
+Python 3 is required for use of these notebooks.
 
 Alternatively, if you would like to execute the notebooks and view them locally, you can clone this repo and do ``python convert.py``. This requires the [nbpages](https://github.com/eteq/nbpages) python package, which you can most easily install by doing ``pip install -e git+https://github.com/eteq/nbpages.git#egg=nbpages``.
 
+Below are instructions for using conda to to create an environment with everything needed to run locally.
+
+### Building Locally with Conda
+
+To quickly create an environment with everything needed to run and convert, please first install Conda or Miniconda to your machine using [Conda Installation Instructions](https://conda.io/projects/conda/en/latest/user-guide/install/index.html).
+
+Once you have Conda installed, complete the following from a command line:
+
+```bash
+git clone https://github.com/spacetelescope/notebooks
+cd notebooks
+conda env create -f environment.yml
+conda activate notebooks_env
+python convert.py
```

Comment:
When I was first converting the notebooks, I had to run `python convert.py --report results.xml` or I would get an error I couldn't get around.  Should that be included in these instructions too (or was that just a me problem)?

Reply:
This was indeed a bug, I am working on a fix now. Thank you for finding this! Of course when adding the option for test reports I extensively tested generating reports but failed to also test that running convert without reporting turned on still worked.